src/Meta/particles_distribution.jl

```diff
@@ -0,0 +1,14 @@
+particles_distribution_meta = Dict(
```

Comment:
What happened here? It seems you have two files in the Meta. One (the generated one) misses the .jl, while the other seems incomplete `" ⋯ 451 bytes ⋯ "ult_nvar, kwargs...) = 0`.